package.json

```diff
@@ -10,6 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "axios": "^0.16.2",
```

Comment:
change this line to `"axios": "^1.7.2", as it's the most recent version and works with your graph and stuff still

Reply:
After you do this, make sure you run `npm i` again.